pkg/awsvpc/parser.go

```diff
@@ -291,6 +296,65 @@ func (rc *AWSresourcesContainer) getSGconfig(
 	return nil
 }
 
+func (rc *AWSresourcesContainer) getNACLconfig(
+	res *vpcmodel.MultipleVPCConfigs,
+	skipByVPC map[string]bool,
+) error {
+	naclLists := map[string][]*commonvpc.NACL{} // map from vpc uid to its nacls
+	for i := range rc.NetworkACLsList {
+		nacl := rc.NetworkACLsList[i]
```

Comment:
```suggestion
	for _, nacl := range rc.NetworkACLsList {
```
When you have a slice of pointers, this is the best syntax